WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.0.8:5000/ (Press CTRL+C to quit)
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.0.8:5000/ (Press CTRL+C to quit)
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.0.8:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [21/Sep/2024 13:22:36] "GET / HTTP/1.1" 200 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.0.8:5000/ (Press CTRL+C to quit)
ERROR:main:Exception on / [GET]
Traceback (most recent call last):
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 1968, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 2097, in make_response
    raise TypeError(
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [21/Sep/2024 13:23:54] "[35m[1mGET / HTTP/1.1[0m" 500 -
ERROR:main:Exception on / [GET]
Traceback (most recent call last):
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 1968, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 2097, in make_response
    raise TypeError(
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [21/Sep/2024 13:24:15] "[35m[1mGET / HTTP/1.1[0m" 500 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.0.8:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [21/Sep/2024 13:27:40] "GET / HTTP/1.1" 200 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.0.8:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [21/Sep/2024 13:30:55] "GET / HTTP/1.1" 200 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.0.8:5000/ (Press CTRL+C to quit)
ERROR:main:Exception on / [GET]
Traceback (most recent call last):
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 1968, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 2097, in make_response
    raise TypeError(
TypeError: The view function did not return a valid response. The function either returned None or ended without a return statement.
INFO:werkzeug:127.0.0.1 - - [21/Sep/2024 13:31:36] "[35m[1mGET / HTTP/1.1[0m" 500 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.0.8:5000/ (Press CTRL+C to quit)
ERROR:main:Exception on / [GET]
Traceback (most recent call last):
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 2117, in make_response
    rv = self.response_class.force_type(rv, request.environ)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/werkzeug/wrappers/response.py", line 263, in force_type
    response = Response(*run_wsgi_app(response, environ))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/werkzeug/test.py", line 1229, in run_wsgi_app
    app_rv = app(environ, start_response)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: str() argument 'encoding' must be str, not function

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 1953, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 1968, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 2125, in make_response
    reraise(TypeError, new_error, sys.exc_info()[2])
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/_compat.py", line 38, in reraise
    raise value.with_traceback(tb)
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/flask/app.py", line 2117, in make_response
    rv = self.response_class.force_type(rv, request.environ)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/werkzeug/wrappers/response.py", line 263, in force_type
    response = Response(*run_wsgi_app(response, environ))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/pc-nerd/All_alx_projects/Alx_Specialization/alx-backend-user-data/0x01-Basic_authentication/.env/lib/python3.12/site-packages/werkzeug/test.py", line 1229, in run_wsgi_app
    app_rv = app(environ, start_response)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: str() argument 'encoding' must be str, not function
The view function did not return a valid response. The return type must be a string, dict, tuple, Response instance, or WSGI callable, but it was a type.
INFO:werkzeug:127.0.0.1 - - [21/Sep/2024 13:32:51] "[35m[1mGET / HTTP/1.1[0m" 500 -
WARNING:werkzeug: * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
INFO:werkzeug: * Running on http://192.168.0.8:5000/ (Press CTRL+C to quit)
INFO:werkzeug:127.0.0.1 - - [21/Sep/2024 13:33:36] "GET / HTTP/1.1" 200 -
